### PR TITLE
doc: add minutes for Aug 30 2016

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: doc/meetings/build-wg-meeting-2016-08-30.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
+# Node.js Foundation Build WG Meeting 2016-08-30
+
+* GitHub issue: https://github.com/nodejs/build/issues/474
+* Meeting video: http://www.youtube.com/watch?v=p6viQ2HyGqk 
+
+## Present
+
+* Hans Kristian Flaatten
+* JoãReis
```
Comment (jbergstroem) [inline doc/meetings/build-wg-meeting-2016-08-30.md]: João Reis

File: doc/meetings/build-wg-meeting-2016-08-30.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+* Johan Bergströ Michael Dawson
```
Comment (jbergstroem) [inline doc/meetings/build-wg-meeting-2016-08-30.md]: missing m and linefeed

File: doc/meetings/build-wg-meeting-2016-08-30.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+## Standup
+
+* Hans Kristian Flaatten (I can go first for once ;)
+  * been given a formal onboarding by Johan :tada:
+  * started working on Ansible playbook for Alpine Linux
+  * looked into adding offline reasons to Jenkins alerts
+    * .Ping timeout., .Low disk space. etc.
+  * looked into moving Jenkins Monitor to Node.s infra
+    * currently on a personal DO droplet & Sendgrid account 
+* JoãReis
```
Comment (jbergstroem) [inline doc/meetings/build-wg-meeting-2016-08-30.md]: name again

File: doc/meetings/build-wg-meeting-2016-08-30.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+  * Wave of windows failures
+  * Release machines with VS2015
+* Johan Bergströ * redeployed f24 instance to run v8 tests
```
Comment (jbergstroem) [inline doc/meetings/build-wg-meeting-2016-08-30.md]: name/newline

File: doc/meetings/build-wg-meeting-2016-08-30.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+    so we can probably leave this on the back burner for now.
+  * semi-related, node serialport might move to nodejs org, 
+    nvm has also made a request.
+        
+* OS X buildbots/ci: call to action #367
+  * giving the sponsor option one more week otherwise will move forward
+    to use foundation money.
+  * We already have sign off to use foundation money to get this resolved.
+
+
```
Comment (jbergstroem) [inline doc/meetings/build-wg-meeting-2016-08-30.md]: redundant extra line

File: doc/meetings/build-wg-meeting-2016-08-30.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+  * Don't want to use the Node build under test to run the converter
+    because if you break Node bad enough you.ll end up not getting
+    any parsed output.
+  * Johan says this can be done on Jenkins master by writing
+    to artifacts files which are sent back to Jenkins for processing 
+
+## Next meeting
+
+September 20th, 8pm UTC
+~
```
Comment (jbergstroem) [inline doc/meetings/build-wg-meeting-2016-08-30.md]: tilde?

File: doc/meetings/build-wg-meeting-2016-08-30.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+    *  we move to produce JUnit from our test suite
+  * many other tasks depends / consumes TAP output
+  * CITGM has had great success with JUnit output
+    * great performance
+    * easier to dig through / debug
+  * Myles volunteers to take a stab at converter to generate junit from the
+    existing tap files
+  * Don't want to use the Node build under test to run the converter
+    because if you break Node bad enough you.ll end up not getting
+    any parsed output.
```
Comment (jbergstroem) [inline doc/meetings/build-wg-meeting-2016-08-30.md]: I think this comment can be removed

File: doc/meetings/build-wg-meeting-2016-08-30.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
...
+* TAP Plugin issues on Jenkins #453
+
+## Standup
+
+* Hans Kristian Flaatten (I can go first for once ;)
+  * been given a formal onboarding by Johan :tada:
+  * started working on Ansible playbook for Alpine Linux
+  * looked into adding offline reasons to Jenkins alerts
+    * .Ping timeout., .Low disk space. etc.
+  * looked into moving Jenkins Monitor to Node.s infra
```
Comment (jbergstroem) [inline doc/meetings/build-wg-meeting-2016-08-30.md]: Node.js

**Comment by Starefossen** [comment]
LGTM with Johan's nit picks.

**Comment by mhdawson** [comment]
@jbergstroem addressed all of your comments.  Let me know if it looks ok now.

**Comment by jbergstroem** [comment]
LGTM

**Comment by mhdawson** [comment]
landed as https://github.com/nodejs/build/commit/1e4941d1b29c0c370fd289ca3bd289f51694c5b1

**Pull request closed without merging**